src/cpu/pred/tage_base.cc

```diff
@@ -1,4 +1,16 @@
 /*
+ * Copyright (c) 2022-2023 The University of Edinburgh
```

Comment:
This file doesn't have any other changes, so don't update the copyright header here.